Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Build deployable artifacts
+FROM ubuntu:latest as plotman-builder
+
+ARG CHIA_BRANCH
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl jq python3 ansible tar bash ca-certificates git openssl unzip wget python3-pip sudo acl build-essential python3-dev python3.8-venv python3.8-distutils apt nfs-common python-is-python3 cmake libsodium-dev g++
+
+RUN echo "cloning ${CHIA_BRANCH}"
+RUN git clone --branch ${CHIA_BRANCH} https://github.com/Chia-Network/chia-blockchain.git \
+&& cd chia-blockchain \
+&& git submodule update --init mozilla-ca \
+&& chmod +x install.sh 
+
+WORKDIR /chia-blockchain
+# Placeholder for patches
+RUN /usr/bin/sh ./install.sh
+
+COPY . /plotman
+
+RUN . ./activate \
+&& pip install -e /plotman \
```

Comment:
If you don't do `-e` (`--editable`) here then you don't have to copy over `/plotman` into the final stage.  It will just be part of the chia environment.  There are a few other references that I think would go away as well.